### DISCUSSION TITLE
Support For Gists

### DISCUSSION THREAD

**Discussion by blarosen95** [community/community#11277] [Mobile] [answered]
It looks like opening a link to a gist (from browser) on Android isn't supported. Even worse is that it attempts to open the GH app but constantly crashes and reopens, making it rather difficult to terminate.

**Comment by mxie** [comment, marked as answer]
Thanks for reaching out to us @blarosen95! Gists are currently not supported on our apps but we may consider it in the future if there's enough demand. Here's an existing discussion about this feature: https://github.com/github/feedback/discussions/3331.

On a side note, the app shouldn't just crash if you try to open a gist. Instead, it should take you to the browser.

Can you tell me what version of the app you have installed?

**Reply by blarosen95** [reply]
Hi @mxie,

It looks like I've got 1.44.0 installed on my Razer Phone 2 (running Android 9, with Nova 7.0.57 if that's of relevance). I do see there's an update, so I've gone ahead and updated while typing this. After updating, I now have v1.51.1 installed. Opening a Google search result in mobile Chrome that links to a gist.github.com page: my user journey continues within the browser instead of attempting to migrate into the GH app, opening a normal repo link in the same fashion brings me into the app as expected.

Looks like the issue aspect of this can be considered resolved. Thank you!
